.ci/load/Jenkinsfile

```diff
@@ -0,0 +1,170 @@
+        string(name: "concurrent_requests", defaultValue: "100", description: "The number of concurrent requests to test with")
+        string(name: "duration", defaultValue: "10m", description: "Test duration in minutes. Max: 280 minutes")
+        // num_of_runs currently unsupported
+        // string(name: "num_of_runs", defaultValue: "1", description: "Number of test runs to execute")
+        text(name: "agent_config", "defaultValue": "", description: "Custom APM Agent configuration. (WARNING: May echo to console. Do not supply sensitive data.)")
+        text(name: "locustfile", "defaultValue": "", description: "Locust load-generator plan")
+        booleanParam(name: "local_metrics", description: "Enable local metrics collection?", defaultValue: false)
+        // End script auto-generation
+    }
+
+    stages {
+        stage('Pre-flight'){
+            steps {
+                echo 'Getting authentication information from Vault'
+                withSecretVault(secret: 'secret/apm-team/ci/bandstand', user_var_name: 'APP_TOKEN_TYPE', pass_var_name: 'APP_TOKEN'){
+                    sh(script: ".ci/load/scripts/start.sh", returnStdout: true)
+                    setEnvVar('SESSION_TOKEN', sh(script: ".ci/load/scripts/start.sh", returnStdout: true).trim())
```

Comment:
Should we mask this SESSION_TOKEN?

Reply:
It's not a secure token. All security is handled via the APP_TOKEN which is a shared secret. The session token is really just an identifier for the session so it shouldn't need to be obscured.

Reply:
I do not think so, it is a TOKEN to coordinate the process, probably we can use BUILD_ID instead is a unique identified per build.

Reply:
Using something like the BUILD_ID would require some changes to the process as it is currently designed. At present, a unique SESSION_ID is generated by the orchestrator and passed back to Jenkins which then uses it for the lifetime of the pipeline. I agree that we could change this to let Jenkins tell the orchestrator which BUILD_ID it wants to use, but I'm not clear on what advantage that would have. Could you please clarify?

Reply:
there is no much benefice to use BUILD_ID, only to not have to generate it, so I think is fine to generate the SESSION_ID